pkgs/by-name/ya/yao-pkg/package.nix

```
@@ -0,0 +1,106 @@
+{
+  autoPatchelfHook,
+  fetchFromGitHub,
+  fetchYarnDeps,
+  fixup-yarn-lock,
+  lib,
+  makeWrapper,
+  nodejs,
+  stdenv,
+  yarn,
+  yarnBuildHook,
+  yarnConfigHook,
+  yarnInstallHook,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "pkg";
+  version = "6.1.1";
+
+  src = fetchFromGitHub {
+    owner = "yao-pkg";
+    repo = pname;
+    rev = "v${version}";
```

(You might need to rebase to latest master for this)
```suggestion
    tag = "v${version}";
```

---

Have applied this